core/tests/protractor_utils/workflow.js

```diff
@@ -20,12 +20,14 @@
 var forms = require('./forms.js');
 var editor = require('./editor.js');
 var general = require('./general.js');
+var ExplorationEditorPage = require('./ExplorationEditorPage');
 var LibraryPage = require('./LibraryPage.js');
 
 // Creates an exploration and opens its editor.
 var createExploration = function() {
   createExplorationAndStartTutorial();
-  editor.exitTutorialIfNecessary();
+  explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+  explorationEditorPage.exitTutorialIfNecessary();
```

Comment:
As an FYI I'm a bit iffy about "exitTutorialIfNecessary()". We probably should just have exitTutorial() and be deterministic about it, calling it only when necessary.

Reply:
Yes! I concur. Renamed to `exitTutorial()`